dataland-data-exporter/src/main/kotlin/org/dataland/datalanddataexporter/services/CsvExporter.kt

```diff
@@ -55,9 +57,14 @@ class CsvExporter(
 
         createDirectories(outputDirectory)
         val transformationRules = readTransformationConfig("./transformationRules/SfdrSqlServer.config")
-        val headers = getHeaders(transformationRules)
+        val activeHeaders = getHeaders(transformationRules)
```

Comment:
active in which sense? perhaps like "current"? also: move down

```suggestion
        val currentHeaders = getHeaders(transformationRules)
```